### DISCUSSION TITLE
E-Mail Tracking

### DISCUSSION THREAD

**Discussion by SdVGits** [plausible/analytics#1993] [Anything Else]
I know little about the technology but maybe as a suggestion for a more distant future.

It would be nice to be able to track openings of email campaigns in addition to websites at some point.

I've been looking for a way to track and compare email opens in a privacy-friendly way for a long time. So far I've only worked with clicks and the recording by Plausible.

Thank you for listening.

Sven

**Comment by metmarkosaric** [comment]
thanks for the feedback @SdVGits! i think this type of privacy friendly email stats would need to come from the newsletter processor itself directly rather than from a third party. from our side, you can use [UTM tags](https://plausible.io/docs/manual-link-tagging) but that will only count clicks and not opens

**Reply by SdVGits** [reply]
Plauible should then have an option to serve uploaded images (e.g. logos) or generate a 1x1px transparent Image that simply tracks the download of those images as opens. (Without personalization to the e-mail address of the recipient) But of course that would be a completely different technique and maybe it's better to keep classic website tracking in focus. Is just an idea.

**Reply by metmarkosaric** [reply]
thanks! perhaps in the future we can expand into email. who knows!? but there's too much to do with web analytics for now! still think email processor is the best for this as they would also know the number of emails sent etc etc. without that, it will never be a complete picture.
